source/beamline/BioXAS/BioXASSSRLMonochromatorMaskState.cpp

```diff
@@ -12,8 +12,8 @@ BioXASSSRLMonochromatorMaskState::BioXASSSRLMonochromatorMaskState(const QString
 
 	// Setup value options.
 
-	addValueOption(Open, "Open", Open);
-	addValueOption(Closed, "Closed", Closed);
+	addValueOption(Closed, "Closed", 1);
+	addValueOption(Open, "Open", 0, true);
```

Comment:
No, because the blades state pv control uses 1 to indicate Closed and 0 to indicate Open. Closed for the mono state must be 0 right now, because the move enums must be listed before read-only enums in AMControl.